src/lib/y2partitioner/widgets/format_and_mount.rb

```diff
@@ -332,6 +337,123 @@ def store
       def items
         SUGGESTED_MOUNT_POINTS.map { |mp| [mp, mp] }
       end
+
+      # The following condintions are checked:
+      # - The mount point is not empty
+      # - The mount point is unique
+      # - The mount point does not shadow a "non shadowable" subvolume
+      def validate
+        return true if !enabled?
+
+        content_validation && uniqueness_validation && subvolumes_shadowing_validation
+      end
+
+    private
+
+      # Validates not empty mount point
+      # An error popup is shown when an empty mount point is entered.
+      #
+      # @return [Boolean] true if mount point is not empty
+      def content_validation
+        return true unless value.empty?
+
+        Yast::Popup.Error(_("Empty mount point not allowed."))
+        false
+      end
+
+      # Validates that mount point is unique in the whole system
+      # An error popup is shown when the mount point already exists.
+      #
+      # @see #duplicated_mount_point?
+      #
+      # @return [Boolean] true if mount point is unique
+      def uniqueness_validation
+        return true unless duplicated_mount_point?
+
+        Yast::Popup.Error(_("This mount point is already in use. Select a different one."))
+        false
+      end
+
+      # Validates that the mount point does not shadow a "non shadowable" subvolume
+      # An error popup is shown when a subvolume is shadowed by the mount point.
+      #
+      # @return [Boolean] true if mount point does not shadow a subvolume
+      def subvolumes_shadowing_validation
```

Comment:
@joseivanlopez convinced me offline. It's different because there is a lot of stuff that still don't exist in the devicegraph... even stuff that is in the devicegraph but we are planning to change its mountpoint.